test/e2e/kubernetes/service/service.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-const commandTimeout = 1 * time.Minute
+const commandTimeout = 3 * time.Minute
```

Comment:
@feiskyer we're seeing something interesting w/ the introduction of 1.16.0-beta.2 (we haven't observed this in earlier 1.16 or other versions of k8s):

- kubectl delete svc <svcname> is taking > 1 minute, which is what our E2E tests have previously configured as the timeout threshold

When I've repro'd locally, the "service deleted" message returns immediately, e.g.:

```
$ kubectl delete svc ingress-nginx-elb
service "ingress-nginx-elb" deleted
```

... and then things stall for 1-2 minutes until control is finally returned to the shell.

We use the same version of kubectl as the version of the API we're testing, so is it possible something has regressed in the kubectl @ version 1.16.0-beta.2 ? Or is it possible that the API itself in -beta.2 is delaying its response to kubectl that would allow it to return control to the shell?

Reply:
have you found something interesting from kube-controller-manager logs? The only changes may related I think is service load balancer finalizer goes to beta (beta means enabled by default) in this release (https://github.com/kubernetes/kubernetes/pull/81691).

Reply:
Will do if we keep seeing this.